cypress/support/commands.js

```diff
@@ -23,3 +23,27 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`);
+});
+
+Cypress.Commands.add('login',(username, password) => {
+    cy.request ({
+        method: 'POST',
+        url: 'https://demoqa.com/Account/v1/Login',
+        body: {
+            userName: 'cat',
+            password: 'Qwerty!1',
+        },
+
+    }).then((response) => {
+        cy.setCookie('token', response.body.token);
+        cy.setCookie('expires', response.body.expires);
+        cy.setCookie('userId', response.body.userId);
```

Comment:
2 test still fails due to this
```suggestion
        cy.setCookie('userID', response.body.userId);
```